javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var noStrings = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] !== typeof 'string') {
```

Comment:
`typeof "string"` will just result in `"string"`, so you can compare against that directly